docs/attributes-registry/messaging.md

```diff
@@ -23,6 +23,7 @@
 |---|---|---|---|
 | `messaging.batch.message_count` | int | The number of messages sent, received, or processed in the scope of the batching operation. [1] | `0`; `1`; `2` |
 | `messaging.client_id` | string | A unique identifier for the client that consumes or produces a message. | `client-5`; `myhost@8742@s8083jm` |
+| `messaging.consumer.group.name` | string | The name of the consumer group message or batch consumer is associated with. | `my-group`; `indexer` |
```

Comment:
```suggestion
| `messaging.consumer.group.name` | string | The name of the consumer group with which a consumer is associated. | `my-group`; `indexer` |
```
Maybe a little bit clearer, it wasn't obvious to me what the value is of mentioning both "message or batch" consumers.